docs/web/authentication.md

```diff
@@ -391,32 +391,46 @@ CodeChecker also supports OAuth-based authentication. The `authentication.method
           * `username`
 
               Field for the username.
-          * `email`
 
-              Field for the email.
-          * `fullname`
+        The `username` field defines what value will be used as the user's unique identifier (i.e. their "username") depending on the OAuth provider.
 
-              Field for the fullname.
+        Currently there are only 2 options for this:
+         * `username`
+         * `email`
+
+        Expected output for each provider:
+
+        `github`
+          * `username` - user's GitHub `login` will be user's identifier
+          * `email` - a request will be sent to fetch the primary email of account to be used as the user's identifier.
+            If it is hidden, an error will be thrown.
+
+        `google`
+          * Only supports `email`, and user's Gmail email will be considered his username.
+
+        `microsoft`
+          * `username` - Company's signum will be the user's identifier.
+          * `email` - an email associated with this Microsoft account will be used as user's identifier.
 ~~~{.json}
 "method_oauth": {
-      "enabled": false,
-      "providers": {
-        "example_provider": {
-          "enabled": false,
-          "client_id": "client id",
-          "client_secret": "client secret",
-          "authorization_url": "https://accounts.google.com/o/oauth2/auth",
-          "callback_url": "http://localhost:8080/login/OAuthLogin/provider",
-          "token_url": "https://accounts.google.com/o/oauth2/token",
-          "user_info_url": "https://www.googleapis.com/oauth2/v1/userinfo",
-          "user_emails_url": "https://api.github.com/user/emails",
-          "scope": "openid email profile",
-          "user_info_mapping": {
-            "username": "email"
-          }
+    "enabled": false,
+    "providers": {
+      "example_provider": {
+        "enabled": false,
+        "client_id": "client id",
+        "client_secret": "client secret",
+        "authorization_url": "https://accounts.google.com/o/oauth2/auth",
+        "callback_url": "http://localhost:8080/login/OAuthLogin/provider",
+        "token_url": "https://accounts.google.com/o/oauth2/token",
+        "user_info_url": "https://www.googleapis.com/oauth2/v1/userinfo",
+        "user_emails_url": "https://api.github.com/user/emails",
+        "scope": "openid email profile",
```

Comment:
I am getting this error at authentication 
User info fetch failed: "jwks_url'

I guess it is missing from the config file

Reply:
But why is it still missing from this authentication md .json example?